docs/modules/ROOT/pages/access.adoc

```diff
@@ -1,672 +1,415 @@
-:ownable-cairo: link:https://github.com/OpenZeppelin/cairo-contracts/blob/release-v0.6.1/src/openzeppelin/access/ownable/library.cairo[Ownable]
-
+:ownable-cairo: link:https://github.com/OpenZeppelin/cairo-contracts/blob/cairo-2/src/access/ownable/ownable.cairo[Ownable]
```

Comment:
why not a release? `cairo-2` is likely to be deleted this week

Reply:
Should I replace it with `release-0.7.0` ?

Reply:
`release-v0.7.0`, yes